src/plugins/plot/plugin.js

```diff
@@ -59,13 +64,48 @@ export default function () {
             },
             priority: 890
         });
+        openmct.types.addType('telemetry.plot.spectral', {
```

Comment:
It will probably be tidier to put all of the non time-domain plotting in a separate plugin. Our current "plot" plugin is going to be focused purely on time-domain plotting.